scripts/mock_data/supplementary_dataset_metadata_response.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "survey_id": "123",
+    "period_id": "ttt",
+    "title": "Supplementary data without repeating items",
+    "sds_schema_version": 1,
+    "sds_published_at": "2022-03-01T12:30:02Z",
+    "total_reporting_units": 2,
+    "schema_version": "v1.0.0",
+    "sds_dataset_version": 1,
+    "filename": "",
+    "dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178"
+  },
+  {
+    "survey_id": "123",
+    "period_id": "ttt",
+    "title": "Supplementary data with repeating items",
+    "sds_schema_version": 1,
+    "sds_published_at": "2022-03-01T12:30:02Z",
+    "total_reporting_units": 2,
+    "schema_version": "v1.0.0",
+    "sds_dataset_version": 1,
```

Comment:
This `sds_dataset_version` should be 2, and the next one 3 to make it realistic data.

Reply:
Done 👍